### PR TITLE
Enhancement/move schain static params to env

### LINKED ISSUE #891: Add "snapshotDownloadTimeout" flag to schain config + remove legacy code
Related to https://github.com/skalenetwork/skaled/issues/1191

Need to add the "snapshotDownloadTimeout" flag to the schain config as an option for skaled. 

Example:
static_schain_params.json:
```
 "schain": {
        "snapshot_interval_sec": 86400,
        "empty_block_interval_ms": 10000,
        "free_contract_deployment": true,
        "snapshot_download_timeout": 18000
```

In skaled config: `snapshotDownloadTimeout`

### LINKED ISSUE #893: Add context predeployed contract
Take context predeployed contract not from skale-node, but in the same way as other predeployeds from:
https://pypi.org/project/context-predeployed/

### PR DESCRIPTION
**Pull request by dmytrotkk**
fixes https://github.com/skalenetwork/skale-admin/issues/893
fixes https://github.com/skalenetwork/skale-admin/issues/891

### CODE REVIEW

File: tools/docker_utils.py [hunk 1]
```diff
@@ -263,7 +264,10 @@ def display_container_logs(
         )
         pretext = f'container {container_name} logs: \n'
         logs = (head_lines + CONTAINER_LOGS_SEPARATOR + tail_lines).decode("utf-8")
-        logger.info(pretext + logs)
+        if to_loggger:
```
Comment (badrogger) [inline tools/docker_utils.py]: Suggest to remove

File: core/schains/config/env_params.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#   -*- coding: utf-8 -*-
```
Comment (badrogger) [inline core/schains/config/env_params.py]: suggest to rename to static_params.py

**Review by badrogger — CHANGES_REQUESTED** [review]

File: tools/docker_utils.py (hunk 1 above)
Comment (dmytrotkk) [reply tools/docker_utils.py]: addressed

File: core/schains/config/env_params.py (hunk 2 above)
Comment (dmytrotkk) [reply core/schains/config/env_params.py]: addressed

**Review by badrogger — APPROVED** [review]

**Pull request merged**